tests/redirects/test_urls.py

```diff
@@ -58,3 +58,32 @@ def test_404_url(base_url):
 def test_url(url, base_url, follow_redirects=False):
     assert_valid_url(url, base_url=base_url,
                      follow_redirects=follow_redirects)
+
+
+@pytest.mark.headless
+@pytest.mark.nondestructive
+@pytest.mark.django_db
+@pytest.mark.parametrize('url', [
+    '/en-US/',
+    '/en-US/firefox/',
+    '/en-US/firefox/all/',
+    '/en-US/firefox/new/',
+    '/en-US/firefox/mobile/',
+    '/en-US/firefox/browsers/',
+    '/en-US/firefox/products/',
+    '/en-US/firefox/accounts/',
+    '/en-US/firefox/channel/desktop/',
+    '/en-US/firefox/lockwise/',
+    '/en-US/firefox/developer/',
+    '/en-US/firefox/installer-help/',
+    '/en-US/firefox/notes/',
+    '/en-US/firefox/whatsnew/all/',
```

Comment:
We host many different /whatsnew pages, which are some of our highest traffic URLs. I don't think it's worth adding them all here, but testing the latest 2 major releases would be nice:

- https://www.mozilla.org/en-US/firefox/78.0/whatsnew/all/
- https://www.mozilla.org/en-US/firefox/77.0/whatsnew/all/

Reply:
Yeah. I just figured that would become outdated very quickly. Not sure how best to make that work well. How do you test those pages normally?

Reply:
We have functional tests for each major release. When we remove templates, we remove the accompanying tests.

Reply:
Looks like none of those pages have any tests in functional at the moment. 76 is the latest test I can find.

Reply:
Sigh, they should have tests :(

the tests for Dev edition and Nightly do exist, they just don't change version to version:

https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/whatsnew/test_whatsnew_developer_70.py
https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/whatsnew/test_whatsnew_nightly_70.py

Can we please just add these URLs so they have some level of coverage?

Reply:
Yeah. I was just looking to see if maybe there was some automated way to discover the tests to run. I could automatically grab all of the firefox URLs and test those I think... Though I may look into running the selenium tests locally instead.